tests/test_utils.py

```diff
@@ -174,8 +174,8 @@ def test_get_static_atoms():
     assert [obj.name for obj in added_atom.objects] == ["block0", "target0"]
 
 
-def test_run_policy_until():
```

Comment:
Let's throw in some tests for `run_policy_until()`? They will be similar to these ones in `test_simulate_policy_until()`.